capture-sdk/sdk/src/main/java/net/gini/android/capture/help/FileImportActivity.java

```diff
@@ -156,6 +168,54 @@ private void setupHomeButton() {
         }
     }
 
+    private void waitForHalfSecondAndShowSnackBar() {
+        new Handler(Looper.getMainLooper()).postDelayed(this::showCustomSnackBar, 500);
+    }
+
+    private void showCustomSnackBar() {
+        ScrollView scrollView = findViewById(R.id.gc_file_import_scrollview);
+
+        Snackbar snackbar = Snackbar.make(scrollView, "", Snackbar.LENGTH_LONG);
+        snackbar.getView().setBackgroundColor(Color.TRANSPARENT);
+
+        Snackbar.SnackbarLayout snackBarLayout = (Snackbar.SnackbarLayout) snackbar.getView();
+        snackBarLayout.setPadding(0, 0, 0, 0);
+
+
+        View view = getLayoutInflater().inflate(R.layout.gc_snackbar_info, null);
+
+        TextView dismissTxt = view.findViewById(R.id.gc_snackbar_dismiss);
+        dismissTxt.setOnClickListener(v -> {
+            if (snackbar.isShown())
+                snackbar.dismiss();
+        });
+
+        snackBarLayout.addView(view, 0);
+
+        snackbar.show();
+
+    }
+
+    @Override
+    protected void onResume() {
+        super.onResume();
+
+        showDarkModeIllustrations(ContextExtKt.isDarkMode(this));
+
+    }
+
+    private void showDarkModeIllustrations(boolean isDarkMode) {
```

Comment:
- Yeah I couldn't make it work by putting illustrations into folders but yeah now I see why haha because the name was different.  First I thought to go programmatically then I've seen the `--night` folders but I didn't change the names to be the same again.